src/README.md

```diff
@@ -0,0 +1,37 @@
+# Grafana Cribl Plugin
+
+Welcome to the Grafana Cribl plugin README. This plugin empowers you to create Grafana data sources pulling data directly from Cribl Search, with the resulting data visualizations offering insights like never before. Let's get into how to set this up.
```

Comment:
great. I would have expected most of this in the top-level README. Is this README at least linked from there?

Reply:
This shows up in the Grafana UI when you click on the Cribl plugin.

Reply:
Added a link & more info to the top-level README.